src/interfaces/filing-interface.ts

```diff
@@ -11,5 +11,8 @@ export interface FilingIF {
   header: HeaderIF,
   incorporationApplication?: any;
   alteration?: AlterationIF;
+  registrarsNotation?: any;
+  registrarsOrder?: any;
+  courtOrder?: any;
```

Comment:
Future state: Add an interface to our shared components for these filings as we have other filings. 
Concerned we will still have some changes to come due to these filings still in development, so i'm certain some changes will happen along the way 👍

Reply:
Agreed. Do you think we need to keep track of this (new ticket or code comment?), or is it something you'll likely to do on the side, lol ?

Reply:
😆 Good question. Can we leave this un resolved for now?
I will most likely just do this on the side. These filings are very small but have been subject to change.
Will be very small to add.